modules/game/src/main/Game.scala

```diff
@@ -323,7 +328,10 @@ case class Game(
     history.lastAction map {
       case d: Uci.Drop          => s"${d.pos}${d.pos}"
       case m: Uci.Move          => m.keys
+      case l: Uci.Lift          => s"^${l.pos}"
```

Comment:
i think this wants to be the same as drop? its not uci its keys which i think are used for working out which squares to highlight